### PR TITLE
Add WardenSwap DEX

### PR DESCRIPTION
**Pull request by chain-l**
Adding wardenswap trades data

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @chain-l 
[Sample wardenswap_optimism_trades]()

File: dbt_project.yml [hunk 1]
```diff
@@ -1425,4 +1425,9 @@ models:
...
       +schema: openxswap
       optimism:
-        +schema: openxswap_optimism
\ No newline at end of file
+        +schema: openxswap_optimism
+
+    wardenswap:
+      +schema: wardenswap
+      optimism:
+        +schema: wardenswap_optimism
```
Comment (jeff-dude) [inline dbt_project.yml:1433]: we're looking to eventually clean up the project file a bit by removing schema definitions there, rather put the schema name in the model config block itself.

do you mind removing modifications to this file and ensuring each new spell has the schema config properly applied?

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: dbt_project.yml (hunk 1 above)
Comment (chain-l) [reply dbt_project.yml]: Done. 
Removed from dbt_project.yml and have added schema in config.

Let me know if there is anything else.

**Review by jeff-dude — APPROVED** [review]
thank you!

**Pull request merged**